README.md

```diff
@@ -110,18 +110,19 @@ Phone numbers can be deserialized from an international format by setting the ty
 
 ### Using `libphonenumber\PhoneNumber` objects in forms
 
-You can use the `tel` form type to create phone number fields. There are two widgets available.
+You can use the `PhoneNumberType` (`tel` for Symfony 2) form type to create phone number fields. There are two widgets available.
```

Comment:
Yeah, I was just trying to not have too many variants in the documentation (and deprecations are shown to the user). Might reword it to `tel for Symfony <= 2.7` though